sklearn/cluster/k_means_.py

```diff
@@ -777,12 +779,14 @@ class KMeans(BaseEstimator, ClusterMixin, TransformerMixin):
         If None, the random number generator is the RandomState instance used
         by `np.random`.
 
-    copy_x : boolean, default True
+    copy_x : boolean, optional
         When pre-computing distances it is more numerically accurate to center
-        the data first.  If copy_x is True, then the original data is not
-        modified.  If False, the original data is modified, and put back before
-        the function returns, but small numerical differences may be introduced
-        by subtracting and then adding the data mean.
+        the data first.  If copy_x is True (defualt), then the original data is
+        not modified, ensuring X is C-contiguous.  If False, the original data
```

Comment:
Double space

Reply:
For consistency I think we can keep the double space here.  I heard that @jnothman was using that somewhere and some talk of lawyer came up, he might have been the first to introduce that in the first place. Or Emacs.

Reply:
The first consistency rule should be grammar rule :). Emacs are adding those when compacting several paragraphs together. Anyway since that it was there before, I let go.